app/helpers/application_helper.rb

```diff
@@ -22,6 +22,11 @@ def title
     end
   end
 
+  def extends_layout(layout, **locals, &block)
```

Comment:
Neat trick adapted from this StackOverflow answer: https://stackoverflow.com/a/29536549

Reply:
Would we be able to remove this helper method and use the block passed to a template render?

```
<%= render template: "layouts/application", locals: {} do %>
  content inside layout here
<% end %>
```

Reply:
I think I tried something like that first and it didn't work, but I'll try it again. I think this is part of what the `replace capture(&block)` is doing in trying to "build up" the content.

Reply:
It doesn't seem to work. I don't know that a block can be passed with a `template` like this. It causes an error:

```
'nil' is not an ActiveModel-compatible object. It must implement :to_partial_path.
```

Reply:
bummer, but thanks for trying!